utils/protobuf/rxsamplesmetadata.proto

```diff
@@ -5,6 +5,9 @@ package rxsamplesmetadata;
 message RxSamplesMetadata {
   uint32 sequence_num = 1;
   uint32 numberofreceivesamples = 2;
-  string shrmemname = 3; // REVIEW #26 this should indicate that it's the location of shared memory?
-  float sequence_time = 4;
+  //string shrmemname = 3; // REVIEW #26 this should indicate that it's the location of shared memory?
+  double sequence_time = 3;
+  double time_zero = 4;
+  double start_time = 5;
```

Comment:
again, the names could be changed here to something like 'buffer_start_time' and 'pulse_start_time'/'sequence_start_time'